universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java

```diff
@@ -33,31 +34,56 @@
 import views.style.Styles;
 
 public class ApplicantLayout extends BaseHtmlLayout {
+
   private static final String CIVIFORM_TITLE = "CiviForm";
 
   private final ProfileUtils profileUtils;
   public final LanguageSelector languageSelector;
+  public final String supportEmail;
 
   @Inject
   public ApplicantLayout(
       ViewUtils viewUtils,
       Config configuration,
       ProfileUtils profileUtils,
-      LanguageSelector languageSelector) {
+      LanguageSelector languageSelector,
+      Config appConfig) {
     super(viewUtils, configuration);
     this.profileUtils = checkNotNull(profileUtils);
     this.languageSelector = checkNotNull(languageSelector);
+    this.supportEmail = checkNotNull(appConfig).getString("support_email_address");
+  }
+
+  private Content renderWithSupportFooter(HtmlBundle bundle, Messages messages) {
+    ContainerTag supportLink =
+        div()
+            .with(
+                text(messages.at(MessageKey.FOOTER_SUPPORT_LINK_DESCRIPTION.getKeyName())),
+                text(" "),
+                a(supportEmail)
+                    .withHref("mailto:" + supportEmail)
+                    .withTarget("_blank")
+                    .withClasses(Styles.TEXT_BLUE_800))
+            .withClasses(Styles.MX_AUTO, Styles.MAX_W_SCREEN_SM, Styles.W_5_6);
+
+    bundle.addFooterContent(supportLink);
+
+    return render(bundle);
   }
 
   @Override
   public Content render(HtmlBundle bundle) {
     bundle.addBodyStyles(ApplicantStyles.BODY);
     String currentTitle = bundle.getTitle();
+
     if (currentTitle != null && !currentTitle.isEmpty()) {
       bundle.setTitle(String.format("%s — %s", currentTitle, CIVIFORM_TITLE));
     } else {
       bundle.setTitle(CIVIFORM_TITLE);
     }
+
+    bundle.addFooterStyles(Styles.MT_24);
```

Comment:
idk if this is the best way to do this, but putting it all the way on the bottom looked easy to miss at least on my big monitor

Reply:
Does giving it a top margin make it look higher up?

Reply:
No it just pushes it away from the main content a bit